Dockerfile

```diff
@@ -138,8 +138,8 @@ COPY --from=docker.io/autonomy/fhs:v0.2.0 / /
 COPY --from=assets /undionly.kpxe /var/lib/sidero/tftp/undionly.kpxe
 COPY --from=assets /undionly.kpxe /var/lib/sidero/tftp/undionly.kpxe.0
 COPY --from=assets /ipxe.efi /var/lib/sidero/tftp/ipxe.efi
-COPY --from=initramfs /initramfs.xz /var/lib/sidero/env/discovery/initramfs.xz
-ADD https://github.com/talos-systems/talos/releases/download/v0.4.1/vmlinuz /var/lib/sidero/env/discovery/vmlinuz
+COPY --from=initramfs /initramfs.xz /var/lib/sidero/env/agent/initramfs.xz
+ADD https://github.com/talos-systems/talos/releases/download/v0.4.1/vmlinuz /var/lib/sidero/env/agent/vmlinuz
```

Comment:
It no longer makes sense to call this "discovery" since discovery isn't the only thing that this image does now. A more generic name would be better. This is a backwards compatible change since the agent and sidero are shipped as a single unit. In fact we can completely change the agent API at any time if we want.